modules/restoring-the-secrets-and-apimanager.adoc

```diff
@@ -0,0 +1,166 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-using-oadp.adoc assembly
+
+
+[id="restoring-the-secrets-and-apimanager_{context}"]
+= Restoring the secrets and APIManager
+
+.Prerequisites
+
+. You have backed up the 3scale Operator.
+. You have backed up mysql and Redis databases.
+
+.Procedure
+
+. Delete the 3scale operator CRDs along with the "threescale" namespace:
++
+[source,terminal]
+----
+$ oc delete project threescale
+
+"threescale" project deleted successfully
+----
++
+NOTE: If restoring on a separate cluster, ensure that {oadp-short} is installed and configured with `nodeAgent` enabled on the destination cluster similar to the source cluster.
+
+. Restore the Operator if you backed up the Operator. Otherwise, install a fresh 3scale operator:
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  creationTimestamp: "2024-09-24T05:43:20Z"
+  finalizers:
+  - restores.velero.io/external-resources-finalizer
+  generation: 6
+  name: operator-installation-restore
+  namespace: openshift-adp
+  resourceVersion: "83988"
+  uid: cbc10cd0-de8c-442d-bf29-40a07be70626
+spec:
+  backupName: operator-install-backup
+  excludedResources:
+  - nodes
+  - events
+  - events.events.k8s.io
+  - backups.velero.io
+  - restores.velero.io
+  - resticrepositories.velero.io
+  - csinodes.storage.k8s.io
+  - volumeattachments.storage.k8s.io
+  - backuprepositories.velero.io
+  itemOperationTimeout: 4h0m0s
+----
+
+. Manually create the `s3-credentials` secret: 
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+       creationTimestamp: null
+       name: s3-credentials
+stringData:
+  AWS_ACCESS_KEY_ID: something
+  AWS_SECRET_ACCESS_KEY: something
+  AWS_BUCKET: something
+  AWS_REGION: us-east-1
+type: Opaque
+----
+
```

Comment:
And define these fields...will the user always use AWS to spin up a cluster, or should there be a more generic example here?
<1> 
<2>
<3>